unstructured/partition/auto.py

```diff
@@ -133,6 +133,7 @@ def partition(
     ssl_verify: bool = True,
     ocr_languages: Optional[str] = None,  # changing to optional for deprecation
     languages: List[str] = ["eng"],
```

Comment:
change default to "auto"

Reply:
Done. Think I copied that from image.py or pdf.py where the default is "eng". Seems to be present in another file or two. Unsure if it's intentional in those places or not

Reply:
Ah you're right, for image and pdf we want a default of english because there's no language detection that occurs (since the text is not readily available/strategy is not fast), only passing through to OCR. Maybe we shouldn't include a default here in auto, so that the default gets set by individual partitioners as auto or eng.

Reply:
I changed the default to `None` and made `languages` optional so it'd be consistent with the other kwargs. 

I have also heard from Matt a couple of times before that we don't want to change the order of the kwargs/want to introduce new kwargs at the end of the parameters in case users are using positional arguments and making `languages` just not have a default value would have made it necessary to put it before all the args with default values.